src/libaktualizr/primary/aktualizr.cc

```diff
@@ -122,6 +122,12 @@ std::future<result::Install> Aktualizr::Install(const std::vector<Uptane::Target
   return api_queue_.enqueue(task);
 }
 
+std::future<bool> Aktualizr::SendManifest(const Json::Value &custom) {
+  auto promise = std::make_shared<std::promise<bool>>();
```

Comment:
Funny that no code analyzer is complaining about an unused variable.

Reply:
It's probably more difficult to flag that realiably in C++, because it could just rely on side effects of constructors and destructors. For example, a mutex guard is typically not referenced outside its declaration.